tests/Behat/features/edit-block-element.feature

```diff
@@ -34,3 +34,12 @@ Feature: Edit elements in the CMS
     When I go to "/admin/pages/edit/show/6"
       Then I should see "Eve's Block"
       But I should not see "Alice's Block"
+
+    Scenario: I can get to the edit form of an inline-editable block
+      And I go to "/admin/pages/edit/show/6"
+      And I wait until I see the ".element-editor__element" element
```

Comment:
I don't think this assertion was meant to be used this way, is there an alternative?

Reply:
Not sure what  you mean here sorry.

Reply:
See refactor in #374

Reply:
Changes required: rebase